ngpvan_api/canvass.py

```diff
@@ -0,0 +1,65 @@
+"""
+ngpvan_api.canvass
+~~~~~~~~~~~~~~~
+This module contains canvass-data-related API calls.
+"""
+
+import json
+from ngpvan_api import base
+
+class NGPVANCanvassAPI(base.NGPVANAPI):
+
+    def post_noncanvass(self, person_id, result_id, input_type_id = None, contact_type_id = None, date_canvassed = None):
+        """We did not actually contact the person - post the Canvass Result code indicating why not"""
+
+        canvass_data = {
+            "canvassContext": {
+                "contactTypeId": contact_type_id,
+                "inputTypeId": input_type_id,
+                "dateCanvassed": date_canvassed
+            },
+            "resultCodeId": result_id,
+            "responses": []
+        }
+
+        result = self.client.post(
+            '%s/people/%d/canvassResponses' % (self.base_url, person_id),
+            data=json.dumps(canvass_data)
+        )
+        return {'results': [result]}
```

Comment:
Does the canvassResponses API return anything useful on POST, like maybe the ID of the created response? Wondering if this could be something like `return {'results': [result], 'response_id`: result.json()}`, which would more closely match other creation functions.